### DISCUSSION TITLE
今天一直无法链接啊，连上马上又掉了，能不能把us的服务器关掉，只要连us的就掉线

### DISCUSSION THREAD

**Discussion by coolshell23** [getlantern/lantern-forum-cn#1013] [General]
今天一直无法链接啊，连上马上又掉了，能不能把us的服务器关掉，只要连us的就掉线

**Comment by MaxMKMax** [comment]
@coolshell23 您好. 请问只是今天发生这种情况而已还已很长时间了?

**Reply by x113773** [reply]
一直是这样，所以能不能把US的节点给关了？

**Reply by MaxMKMax** [reply]
@x113773 我帮您反馈一下.

**Comment by MaxMKMax** [comment]
@coolshell23 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@coolshell23 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
